content/en/monitors/monitor_types/forecasts.md

```diff
@@ -62,6 +62,8 @@ Datadog automatically sets the **Advanced** options for you by analyzing your me
 * We recommend using larger intervals between points to avoid having noise influence the forecast too much.
 * The number of deviations controls the width of the range of forecasted values. A value of 1 or 2 should be large enough to accurately forecast most "normal" points.
 
+Then, you can choose if the monitor requires a full window of data for evaluation. Setting it to "Requires" might result in evaluations getting skipped. Indeed, evaluations will get skipped if the data is sparse or if the past data in the evaluation window has some missing values. 
```

Comment:
```suggestion
Then, you can choose if the monitor requires a full window of data for evaluation. Setting it to "Requires" will force the monitor to ignore (i.e., show "No Data" as the monitor state) any series that doesn't have data going back all the way to the start of the time range shown in the Evaluation Window graph.
```

"skipped" is a backend term that I don't think is exposed to the user anywhere. We just show "No Data" on the status page.

Let's not talk about sparsity here because it's a bit confusing. If only every other interval has a point, for example, the monitor will evaluate, skip, evaluate, skip, etc. based on whether the first point in the window has data or not. I think the starting point of the series is key point. That is, new hosts will be ignored.